app/lib/getFileIcon/index.js

```diff
@@ -1,3 +1,6 @@
 const empty = () => Promise.reject()
-const getFileIcon = process.platform === 'darwin' ? require('./mac') : empty
+const getFileIcon = process.platform === 'darwin' ? require('./mac') : (
```

Comment:
I'd change it to `if/else if/else`. Ternary operator looks good only with two branches:)